tests/e2e/support/pages/keyGenerator.js

```diff
@@ -0,0 +1,15 @@
+const keyGenerator = {
+  header: '.header > .label',
+  logo: '.logo',
+  title: 'h2',
```

Comment:
it's very risky to use as a selector an html element. Use CSS class instead of something else but more specific than this one.